cognite/client/data_classes/datapoints.py

```diff
@@ -78,11 +81,14 @@
     "duration_uncertain",
     "interpolation",
     "max",
+    "max_datapoint",
     "min",
+    "min_datapoint",
     "step_interpolation",
     "sum",
     "total_variation",
 ]
+_OBJECT_AGGREGATES: frozenset[Literal["maxDatapoint", "minDatapoint"]] = frozenset({"maxDatapoint", "minDatapoint"})
```

Comment:
Probably a worse name but `_DICT_AGGREGATES`? Took me longer than it should to understand the point of this one.

Reply:
The API calls them object aggregates, hence I used the same.

Reply:
make sense